test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -325,21 +326,23 @@ public void Dispose()
         /// <summary>
         /// Compare the contents of an output file with its baseline.
         /// </summary>
-        protected bool CheckEquality(string dir, string name, string nameBase = null, int digitsOfPrecision = DigitsOfPrecision)
+        protected bool CheckEquality(string dir, string name, string nameBase = null, int? digitsOfPrecision = null)
         {
-            return CheckEqualityCore(dir, name, nameBase ?? name, false, digitsOfPrecision);
+            int precision = digitsOfPrecision ?? (Environment.Is64BitProcess ? DigitsOfPrecision_x64 : DigitsOfPrecision_x86);
+            return CheckEqualityCore(dir, name, nameBase ?? name, false, precision);
         }
 
         /// <summary>
         /// Check whether two files are same ignoring volatile differences (path, dates, times, etc).
         /// Returns true if the check passes.
         /// </summary>
-        protected bool CheckEqualityNormalized(string dir, string name, string nameBase = null, int digitsOfPrecision = DigitsOfPrecision)
+        protected bool CheckEqualityNormalized(string dir, string name, string nameBase = null, int? digitsOfPrecision = null)
         {
-            return CheckEqualityCore(dir, name, nameBase ?? name, true, digitsOfPrecision);
+            int precision = digitsOfPrecision ?? (Environment.Is64BitProcess ? DigitsOfPrecision_x64 : DigitsOfPrecision_x86);
```

Comment:
Seems that `int precision = digitsOfPrecision ?? (Environment.Is64BitProcess ? DigitsOfPrecision_x64 : DigitsOfPrecision_x86);` could be pulled out into a property like `public int DigitsOfPrecision => digitsOfPrecision ?? (Environment.Is64BitProcess ? DigitsOfPrecision_x64 : DigitsOfPrecision_x86);` rather than repeated